packages/ui/src/containers/ChangePasswod/index.tsx

```
@@ -0,0 +1,53 @@
+import React from 'react'
+import styles from './changePassword.module.scss'
+import { useHistory } from 'react-router-dom'
+import TextField from '../../components/TextField'
+import { useState } from 'react'
+import { getWalletIndex, updateWalletName } from '../../services/channels'
+import { isSuccessResponse, Routes } from '../../utils'
+
+const ChangePassword = () => {
+  const [name, setName] = useState('')
+  const history = useHistory()
+
+  const handleInput = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setName(e.target.value)
+  }
+
+  const handleBack = () => {
+    history.push(Routes.Main)
+  }
+
+  const handleSubmit = () => {
+    getWalletIndex().then(res => {
+      if (isSuccessResponse(res) && res.result.current) {
+        const id = res.result.current
+        updateWalletName({ id, name }).then(res => {
+          if (isSuccessResponse(res)) {
+            history.push(Routes.Main)
+          } else {
+            throw new Error(res.message)
```

The error is not handled.